source/documentation/monitoring-an-app/how-to-use-cloudwatch-datasource.html.md.erb

```diff
@@ -1,22 +1,22 @@
 ---
 title: Using the CloudWatch data source in Grafana 
-last_reviewed_on: 2020-02-14
+last_reviewed_on: 2020-02-17
 review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
 
 ### Introduction
 
-Cloud-platform team added [CloudWatch data source][CloudWatch-datasource] in Grafana, using this [CloudWatch data source][CloudWatch-datasource] you can build dashboards for your CloudWatch metrics.
+Cloud-platform team added [CloudWatch data source][CloudWatch-datasource] in Grafana. To build dashboards for your [CloudWatch][cloudwatch-metrics] metrics, you can use the [CloudWatch data source][CloudWatch-datasource].
```

Comment:
I think this part can be removed

> Cloud-platform team added [CloudWatch data source][CloudWatch-datasource] in Grafana.

Because it's a reference document, we only need to describe the current state, so how we got there (CP team added...) isn't useful.

Reply:
Thanks @digitalronin, updated the user guide now